src/test/java/eu/yals/test/utils/elements/VaadinElement.java

```diff
@@ -9,58 +9,58 @@
 import org.openqa.selenium.By;
 
 /**
- * Test methods for Vaadin elements
+ * Test methods for Vaadin elements.
  *
  * @since 2.7
  */
 @Slf4j
 public class VaadinElement extends YalsElement {
-  private final TestBenchElement testBenchElement;
+    private final TestBenchElement testBenchElement;
 
-  public static VaadinElement wrap(TestBenchElement element) {
-    return new VaadinElement(element);
-  }
-
-  public VaadinElement(TestBenchElement element) {
-    super(element);
-    this.testBenchElement = element;
-  }
+    public static VaadinElement wrap(final TestBenchElement element) {
+        return new VaadinElement(element);
+    }
 
-  public void inputShouldBeEmpty() {
-    Assert.assertNotNull("No such element found", testBenchElement);
-    Assert.assertTrue("Input is not TextField", testBenchElement instanceof TextFieldElement);
+    public VaadinElement(final TestBenchElement element) {
+        super(element);
+        this.testBenchElement = element;
+    }
 
-    TextFieldElement element = (TextFieldElement) testBenchElement;
-    Assert.assertTrue("Input is not empty", StringUtils.isBlank(element.getValue()));
-  }
+    public void inputShouldBeEmpty() {
+        Assert.assertNotNull("No such element found", testBenchElement);
+        Assert.assertTrue("Input is not TextField", testBenchElement instanceof TextFieldElement);
 
-  public void errorTextHas(String phrase) {
-    Assert.assertNotNull("No such element found", testBenchElement);
-    Assert.assertTrue(
-        "Element is not NotificationElement", testBenchElement instanceof NotificationElement);
+        TextFieldElement element = (TextFieldElement) testBenchElement;
+        Assert.assertTrue("Input is not empty", StringUtils.isBlank(element.getValue()));
+    }
 
-    NotificationElement notification = (NotificationElement) testBenchElement;
-    String notificationText = getNotificationText(notification);
-    Assert.assertTrue(
-        String.format("Error text: %s does not contain phrase: %s", notificationText, phrase),
-        StringUtils.containsIgnoreCase(notificationText, phrase));
-  }
+    public void errorTextHas(String phrase) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter phrase should be final.](https://app.codacy.com/gh/yadevee/yals/pullRequest?prid=5409395)